ON-15040: Add onload-must-gather image for log collection (#41)
Xilinx-CNS/kubernetes-onload: README.md

## Troubleshooting

> Onload comes with a troubleshooting container with pre-installed tools like `onload_stackdump`. This troubleshooting container is suitable for interactive and automated use.

Reviewer: ```suggestion
The Onload Diagnostics container image includes tools such as `onload_stackdump`. This troubleshooting container is suitable for interactive and automated use.
```

0         -               4159878
```

### Automated

> Collect a log bundle with the `must-gather` tool:

Reviewer: ```suggestion
Collect a log bundle via the OpenShift `must-gather` tool:
```
